docs/reference/ilm/ilm-index-lifecycle.asciidoc

```diff
@@ -36,6 +36,17 @@ the index must be older than the minimum age of the next phase.
 The minimum age defaults to zero, which causes {ilm-init} to move indices to the next phase
 as soon as all actions in the current phase complete. 
 
+If an index has unallocated shards and the cluster is in the _yellow_ state, 
```

Comment:
It may be helpful to disambiguate between this and the general cluster state.
```suggestion
If an index has unallocated shards and the <<cluster-health,cluster health status>> is yellow, 
```